picoscope/picobase.py

```diff
@@ -207,6 +207,10 @@ def setChannel(self, channel='A', coupling="AC", VRange=2.0,
         the allowed values. This will change in furture version where we will
         find the next largest range to accomodate the desired range.
 
+        Note that if you set a channel to use AC coupling, you may need
+        to make a "dummy call" to runBlock, or the first batch of data
+        returned via getData* may be inaccurate.
```

Comment:
Can you add cross links to the issue on github and the upstream issue

Reply:
Done (I think this is what you mean by cross-link to the issue on github?)

There's no upstream issue per se, but I've added a link to the original Picotech forum post about this into the documentation.